solvers/rmsprop_torch.py

```diff
@@ -16,6 +16,7 @@ class Solver(TorchSolver):
         'lr': [1e-3],
         'rho': [0.99, 0.9],
         'momentum': [0, 0.9],
+        'coupled_weight_decay': [0.0, 0.02],
```

Comment:
```suggestion
        'weight_decay': [0.0, 0.02],
```

Reply:
I would rather stick with coupled weight decay to clearly mark the difference with the decoupled weight decay, raising awareness that there is a difference between the 2.